<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Title/ICO.ico</normaloff>:/Title/ICO.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
	color: rgb(0, 0,0);
	background-color: rgba(85, 255, 255, 0);
}

#Settings {
	 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0,
                                   stop: 0 #333333, stop: 1 #777777);
 	border: 1px solid gray;
 	border-radius: 4px;
}
#treeWidgetSettings {
	 background-color: #e9e9e9;
}
#lineEditFindSettings {
	 background-color: #e9e9e9;
}
#scrollAreaWidgetContents {
	 background-color: #e9e9e9;
}
#scrollAreaWidgetContents_2 {
	 background-color: #e9e9e9;
}
#scrollAreaWidgetContents_3 {
	 background-color: #e9e9e9;
}
#scrollAreaWidgetContents_4 {
	 background-color: #e9e9e9;
}
#scrollAreaWidgetContents_5 {
	 background-color: #e9e9e9;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <item row="1" column="0">
    <widget class="QTreeWidget" name="treeWidgetSettings">
     <property name="minimumSize">
      <size>
       <width>220</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="indentation">
      <number>25</number>
     </property>
     <property name="animated">
      <bool>false</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <item>
      <property name="text">
       <string>Основные</string>
      </property>
      <item>
       <property name="text">
        <string>Внешний вид</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Клавиатура</string>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>Синхронизация</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Сервисы</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Системные</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnReset">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>По умолчанию</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>311</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnOk">
        <property name="text">
         <string>ОК</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnApply">
        <property name="text">
         <string>Применить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="lineEditFindSettings">
     <property name="minimumSize">
      <size>
       <width>220</width>
       <height>27</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>27</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#lineEditFindSettings {
 	border: 1px solid gray;
}</string>
     </property>
     <property name="placeholderText">
      <string>Найти настройки</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QStackedWidget" name="stackedWidgetSettings">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="pageAppearance">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QScrollArea" name="scrollArea">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>429</width>
            <height>301</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Тема:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>250</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboBoxTheme">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Темная</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>209</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageKeyboard">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>429</width>
            <height>301</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="5" column="0" colspan="2">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <widget class="QKeySequenceEdit" name="keySequenceEditSync">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string>Синхронизировать всё: </string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QKeySequenceEdit" name="keySequenceEditSyncAll">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="Line" name="line_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string>Синхронизировать: </string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="Line" name="line_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_11">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string>Войти: </string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QKeySequenceEdit" name="keySequenceEditSignIn">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageSync">
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>429</width>
            <height>301</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>27</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel:disabled{color:grey;}</string>
             </property>
             <property name="text">
              <string>Максимальный размер данных для синхронизации (МБ)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="lineEditMaxSyncSize">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>500</string>
             </property>
             <property name="placeholderText">
              <string>в пикселях</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>136</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="0" colspan="4">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBoxAutoSync">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Автоматическая синхронизация</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageServices">
      <layout class="QGridLayout" name="gridLayout_8">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea_4">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>199</width>
            <height>41</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="1" column="0" colspan="2">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QCheckBox" name="checkBoxAutoSignIn">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Автоматический вход в сервис</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageSystem">
      <layout class="QGridLayout" name="gridLayout_10">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea_5">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>429</width>
            <height>301</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="checkBoxStartUp">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Запускать вместе с Windows</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxStartUpMinimized">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Запускать свернутой</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxBackgroundWork">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Разрешить работать в фоне</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
